### ISSUE TITLE
Druid - Spell - Faerie Swarm

### ISSUE THREAD

**Issue by Kwyzard** [HellscreamWoW/Tracker#1336]
http://www.wowhead.com/spell=106707/faerie-swarm
"Grants an improved version of Faerie Fire that also reduces the target's movement speed by 50% for 15 sec.

This talent replaces Faerie Fire."

No reduction of target movement
Target can still stealth 
Buff last for 5 min on target 

And i don't understand what means last sentence but we only have the Faerie Fire but it apllie the Faerie Swarm effect, don't know if we should have a spell called Faerie Swarm in your spell library or not

**Comment by silverjade** [comment]
Yeah, if you choose Faerie Swarm talent, it should replace your Faerie Fire ability with an ability called Faerie Swarm, that prevents stealth + slows the target for 15 sec.
